src/components/BoundPiecesList/constants.js

```diff
@@ -0,0 +1,31 @@
+import { FormattedMessage } from 'react-intl';
+
+export const PIECE_COLUMNS = {
+  displaySummary: 'displaySummary',
+  chronology: 'chronology',
+  copyNumber: 'copyNumber',
+  enumeration: 'enumeration',
+  receiptDate: 'receiptDate',
+  barcode: 'barcode',
+  remove: 'remove',
+};
+
+export const PIECE_COLUMN_MAPPING = {
+  [PIECE_COLUMNS.barcode]: <FormattedMessage id="ui-receiving.piece.barcode" />,
+  [PIECE_COLUMNS.displaySummary]: <FormattedMessage id="ui-receiving.piece.displaySummary" />,
+  [PIECE_COLUMNS.chronology]: <FormattedMessage id="ui-receiving.piece.chronology" />,
+  [PIECE_COLUMNS.copyNumber]: <FormattedMessage id="ui-receiving.piece.copyNumber" />,
+  [PIECE_COLUMNS.enumeration]: <FormattedMessage id="ui-receiving.piece.enumeration" />,
+  [PIECE_COLUMNS.receiptDate]: <FormattedMessage id="ui-receiving.piece.receiptDate" />,
```

Comment:
lets add these translations to ui-inventory translations to not rely on a module that is not listed in `package.json`